src/hooks/NftRewards.ts

```diff
@@ -22,10 +20,12 @@ async function getRewardTierFromIPFS({
   tier: JB721TierParams
   index: number
 }): Promise<NftRewardTier> {
-  const url = restrictedIpfsUrl(decodeEncodedIPFSUri(tier.encodedIPFSUri))
+  const tierCid = decodeEncodedIPFSUri(tier.encodedIPFSUri)
+  const url = publicIpfsUrl(tierCid)
 
-  const response = await axios.get(url)
+  const response = await ipfsGet(url)
```

Comment:
get the NFT reward tier metadata using the infura endpoint (with the Authorization header)